scss/_sjc-header.scss

```diff
@@ -100,18 +116,84 @@ body .omnibar .dropdown-menu  {
 	font-size: 20px;
 	line-height: 1.2rem;
 	display: inline-block;
-	padding-left: 5px;
+	padding-left: 8px;
 	color: lighten($primary, 30%);
 	text-decoration: none;	
 }
 
 .site-title em {
 	font-style: normal;
 }
+
+
+/* Dropdown Menu */
+
+.dropdown-menu {
+	top: 125%;
+	padding: 1.5rem 0;
+	color: $white;
+	background: lighten($primary, 10%) !important;
+}
+
+.nav-flex-fill {
+	padding: 0 1.25rem;
+	flex: 1 1 auto !important;
+	min-width: 225px;
+	border-right: 2px solid darken($primary, 5%);
+}
+
+.nav-flex-fill:first-child {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+}
+
+.nav-flex-fill:last-child {
+	border: none;
+}
+
+.dropdown-item {
+	color: $white;
+	padding: .4rem .5rem;
+	min-width: 10rem;
+	flex: 1 1 auto;
+	border-bottom: solid $primary .5px;
+	&:last-child {
+		border: 0;
+	}
+}
+
+.dropdown-main-options {
+	font-weight: 600;
+	border-bottom: 0;
+}
+
+#icon-nav {
+	max-height: 35px;
+	padding: 0 1rem 0 0;
+}
+
+.profile {

```

Comment:
More descriptive name here. If you want a really verbose but accurate suggestion, `.profile-menu-dropdown-item`.